app/templates/components/bootstrap-modal.hbs

```diff
@@ -2,7 +2,6 @@
   <div class="modal-dialog">
     <div class="modal-content box box-primary">
       <div class="modal-header">
-        <AppUi::Button type="button" class="close" data-dismiss="modal" aria-hidden="true">{{t "views.app.components.modal.times"}}</AppUi::Button>
```

Comment:
Oh, no. We need to keep the button, the issue was with the translation, instead of cross button it was showing &times.

Reply:
There is already a close button, present in modal. So is there any need for cross as well?

Reply:
It was added for custom styling, making it more coherent with the rest of the application. 

What does the default look like, can you please share a snapshot?

Reply:
Sure! Please find below the snapshot for the modal (without the cross button).

![image](https://github.com/user-attachments/assets/bbe6e512-f5ca-4e06-ba8a-10f7fd47eefc)

Reply:
I thought you were talking about a default cross button, but I see you were talking about the close button. 

We still need the cross button, I know it adds redundancy, but it was added as people are used to seeing a close button on the top whether there is a close button in the form or not. Users tend to associate the close button with the form and cross button with modal. 

So, if a user just accidently or otherwise clicked on a button to open a modal but want to dismiss the modal rather than closing the form, they will look for the cross on the top right.

Reply:
Updated with the changes. Resolving this conversation now!